### PR TITLE
fix: Fix security issue 

### PR DESCRIPTION
**Pull request by adkumar1**
- There is an security issue CVE-2023-33202 . This PR will be resolve it.

### CODE REVIEW

**Comment by dvasunin** [comment]
@amoldashwant @sachinargade123 please review

File: DEPENDENCIES [hunk 1]
```diff
@@ -21,9 +21,12 @@ maven/mavencentral/com.google.guava/listenablefuture/9999.0-empty-to-avoid-confl
 maven/mavencentral/com.google.j2objc/j2objc-annotations/2.8, Apache-2.0, approved, clearlydefined
 maven/mavencentral/com.google.protobuf/protobuf-javalite/3.22.3, BSD-3-Clause, approved, clearlydefined
 maven/mavencentral/com.goterl/lazysodium-java/5.1.1, MPL-2.0, approved, #10952
+maven/mavencentral/com.goterl/resource-loader/2.0.1, , restricted, clearlydefined
```
Comment (almadigabor) [inline DEPENDENCIES:24]: This is marked as restricted with no issue number. You need to open an IP issue ticket with the Dash tool. Do you have a committer or should I do it for you?

Comment (dvasunin) [reply DEPENDENCIES]: somehow this dependency has disappeared from the list after update

Comment (almadigabor) [reply DEPENDENCIES]: Right and I also don't see a reference in the pom.xml. I'll mark this as resolved.

**Review by almadigabor — APPROVED** [review]
LGTM

**Pull request merged**